CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Update to v2.21.0 of the NewRelic Terraform Provider
```

Comment:
Is the title of the PR wrong or should this be v2.20.0?
```suggestion
* Update to v2.20.0 of the NewRelic Terraform Provider
```

Reply:
hey! Great catch :) The title of the PR didn't update when I pushed the latest commit for the v2.21.0 upgrade